client/src/research/LoginPage.css

```diff
@@ -0,0 +1,21 @@
+.LoginView {
+  width: 100%;
+  margin: 10px;
+  display: block;
+}
+
+.LoginView-analysis {
+  margin: 20px;
+  margin-bottom: 100px;
+}
+
+.Block {
```

Comment:
The convention for CSS names here is to prefix each class name with the component name.  So this would be `LoginPage-Block`.  This is because the CSS class names are actually global, so if another component had `.SubmitButton` or `.Block` they would conflict.  Adding the prefixes explicitly prevents this.

Reply:
gotcha. I had forgotten about that. Just cleaned up this css file and updated LoginPage.js